contrib/test/crio-integration-playbook.yaml

```diff
@@ -215,31 +215,31 @@
     args:
       chdir: /root/src/github.com/kubernetes-incubator/cri-o/
   - name: run integration tests RHEL
-    shell: 'CGROUP_MANAGER=systemd STORAGE_OPTS="--storage-driver=overlay2 --storage-opt overlay2.override_kernel_check=1" make localintegration 2>&1 > testout.txt'
+    shell: 'CGROUP_MANAGER=cgroupfs STORAGE_OPTS="--storage-driver=overlay2 --storage-opt overlay2.override_kernel_check=1" make localintegration 2>&1 > testout.txt'
```

Comment:
There's an issue with systemd and runc Mrunal is currently investigating, we'll switch back to systemd cgroup management as soon as we find a fix